app/cli/src/main/scala/org/bitcoins/cli/CliReaders.scala

```diff
@@ -44,6 +45,14 @@ object CliReaders {
       val reads: String => Bitcoins = str => Bitcoins(BigDecimal(str))
     }
 
+  implicit val satoshisPerVirtualByteReads: Read[SatoshisPerVirtualByte] =
+    new Read[SatoshisPerVirtualByte] {
+      val arity: Int = 1
+
+      val reads: String => SatoshisPerVirtualByte = str =>
+        SatoshisPerVirtualByte(Satoshis(BigInt(str)))
```

Comment:
doesn't this clash with your other PR that moves things to Doubles?

Reply:
Yes